easybuild/easyblocks/c/cp2k.py

```diff
@@ -97,6 +97,7 @@ def extra_options():
             'extradflags': ['', "Extra DFLAGS to be added", CUSTOM],
             'ignore_regtest_fails': [False, ("Ignore failures in regression test "
                                              "(should be used with care)"), CUSTOM],
+            'library': [True, "Also build CP2K as a library", CUSTOM],
```

Comment:
@ocaisa By enabling this by default, this has to work on all existing `CP2K` easyconfigs, which includes some very old ones...

Maybe `False` is a better default, and people can just build it when they need it? We can start enabling it for recent `CP2K` easyconfigs, of course...